web_field_tooltip/static/src/js/web_field_tooltip.js

```diff
@@ -0,0 +1,131 @@
+/* Copyright 2023 ACSONE SA/NV
+   Copyright 2019 TODAY Serpent Consulting Services Pvt. Ltd.
+   License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
+
+odoo.define("web_field_tooltip.FieldTooltip", function(require) {
+    "use strict";
+
+    var core = require("web.core");
+    var FormRenderer = require("web.FormRenderer");
+    var FormController = require("web.FormController");
+    var field_utils = require("web.field_utils");
+    var rpc = require("web.rpc");
+    var _t = core._t;
+
+    FormRenderer.include({
+        /**
+         * @override
+         */
+        _renderTagLabel: function(node) {
+            var self = this;
+            var $result = this._super.apply(this, arguments);
+            var fieldName = node.tag === "label" ? node.attrs.for : node.attrs.name;
+
+            // Check if there's any tooltip that must be rendered for the given view
+
+            rpc.query({
```

Comment:
I don't think it's a good approach to make the query here, as it will create one query per field each time we render it (even using the pagination and even if there is no tooltip for the current model). I think the tooltip per fields should be retrieved only one time, when requesting the views to be displayed

Reply:
you're right. i'll try to improve this

Reply:
here's a solution that doesn't satisfy me fully: https://github.com/OCA/web/pull/2553/commits/5fbb022a00dc317f57d400b5d1fad156ed6120df

problem is if you don't reload the page and go back directly to form view using breadcrumbs, you'll have old tooltips (new tooltips won't be there and deleted tooltips will still be there)

Reply:
In my opinion this is not a problem as there are a lot of things that are not refreshed without doing a full reload. Can you just store the tooltips on the FormRenderer instead of ```$el``` ? Even if it works, it seems weird to me to store data on the DOM element. Otherwise I think we should inherit from load_views to load the tooltips, and then use it in the FormRenderer, but I'm not sure that we could inherit easily from odoo code to store the retrieved tooltips.